testsuite/regressiontests/ticket-659-copyright.py

```diff
@@ -38,6 +39,15 @@
     'CMakeFiles',
 ]
 
+# match all file names against these regular expressions. if a match
+# is found the file is excluded from the check
+exclude_file_patterns = [
+    '\.#.*',
+    '#.*',
+    '.*~',
+]
+exclude_file_regex = [re.compile(pattern) for pattern in exclude_file_patterns]
```

Comment:
I think it would be more elegant to have a single regular expression using the or operator, i.e., something like `\.#.*|#.*|.*~`. Maybe also add `.bak`.